cadwyn/applications.py

```diff
@@ -36,7 +38,7 @@
 logger = getLogger(__name__)
 
 
-@dataclasses.dataclass(slots=True)
```

Comment:
I think, we can simply have a dict with arguments defined as a global constant that only has slots if the version is right:

```python
SLOTS = {"slots": True}
@dataclasses.dataclass(**slots)
```